### DISCUSSION TITLE
I am trying to check to see if my service is running after restart

### DISCUSSION THREAD

**Discussion by galtline** [liquidctl/liquidctl#421] [Q&A]
After restart I ran the following

$ systemctl status liquidcfg.service

liquidcfg.service - AIO startup service
     Loaded: loaded (/etc/systemd/system/liquidcfg.service; enabled; vendor preset: enabled)
     Active: inactive (dead) since Sun 2022-02-06 16:07:38 PST; 11min ago
    Process: 685 ExecStart=/usr/local/bin/liquidctl initialize all (code=exited, status=0/SUCCESS)
    Process: 2988 ExecStart=/usr/local/bin/liquidctl --match H115i set fan speed 20 20 30 30 40 60 55 80 70 100 (code=exited, status=0/SUCCESS)
   Main PID: 2988 (code=exited, status=0/SUCCESS)

Feb 06 16:07:37 bol systemd[1]: Starting AIO startup service...
Feb 06 16:07:38 bol  liquidctl[685]: Corsair Hydro H115i Pro XT
Feb 06 16:07:38 bol liquidctl[685]: └── Firmware version    1.1.22
Feb 06 16:07:38 bol systemd[1]: liquidcfg.service: Succeeded.
Feb 06 16:07:38 bol systemd[1]: Finished AIO startup service.

What does this mean:   Active: inactive (dead) since Sun 2022-02-06 16:07:38 PST; 2min 13s ago

Also what is the best way to test to make sure the fans are working with load.

**Comment by jonasmalacofilho** [comment]
> What does this mean: Active: inactive (dead) since Sun 2022-02-06 16:07:38 PST; 2min 13s ago

It means that the service is enabled, but not currently running.  Which is what you expect.

The important lines are:

```
Feb 06 16:07:38 bol systemd[1]: liquidcfg.service: Succeeded.
Feb 06 16:07:38 bol systemd[1]: Finished AIO startup service.
```

as well as the absence of errors.

> Also what is the best way to test to make sure the fans are working with load.

Something like:

```
# on one terminal window
$ watch -d -n 1 liquidctl status
```

```
# on another terminal window
$ stress -c $(nproc)
```

Alternatively you can do without `watch` by simply running `liquidctl status` in a loop, and replace `stress` which some other CPU-heavy task you're used to running.

Also note that the fan/pump profiles run on the device and are always based on the internal liquid temperature probe.  Take a look at the [documentation section for Pro XT coolers and liquidctl 1.5.1](https://github.com/liquidctl/liquidctl/blob/v1.5.1/docs/corsair-platinum-pro-xt-guide.md).

---

P.S. Will move this to the Discussions tab (Q&A section).

P.P.S. You don't need to open new issues/discussions for related subjects (in this case: general usage questions) or just because a previous issue/discussion has been closed.  An issue being closed is really just a way for us to organize what we need to tackle (generally, issues are things we still need to change something to address).  Also, many separate issues/discussions for the same thread create a bit of noise for us.
